src/Workspaces/Core/Portable/Remote/RemoteHostClient.cs

```diff
@@ -72,6 +72,11 @@ private void OnStatusChanged(bool started)
             StatusChanged?.Invoke(this, started);
         }
 
+        public static string CreateClientId(string prefix)
+        {
+            return $"VS ({prefix}) ({Guid.NewGuid().ToString()})";
```

Comment:
minor: Why not put the process.getcurrentprocess here so that prefix can never be anything other than id of current process?

Reply:
process is not availabe in .net standard. I need to add additional nuget to get it and didn't want to do so.